apps/fulfillment/sw/app.ts

```diff
@@ -1,8 +1,13 @@
+import { bootstrap } from '@spryker-oryx/application/service-worker';
 import { appBuilder } from '@spryker-oryx/application';
 import { offlineServiceWorkerFulfillmentFeatures } from '@spryker-oryx/presets/fulfillment';
 import { fallbackEnv } from '../src/fallback-env';
 
-export const app = appBuilder()
+appBuilder()
   .withEnvironment({ ...fallbackEnv, ...process.env })
   .withFeature(offlineServiceWorkerFulfillmentFeatures())
-  .create();
+  .create()
+  .then(() => console.debug('Service worker app started!'))
+  .catch(console.error);
```

Comment:
do we need this?

Reply:
It was there in the original code and I just thought to keep it. But it doesn't show up in the logs anyways despite the service worker registering successfully.